EE
======================================================================
ERROR: test_server_unwatch (__main__.TestServer)
watch a currently watched job.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "server_watch_test.py", line 33, in test_server_unwatch
    stdoutput, stderror, status = _exec(['./jobclient8', 'watch {}'.format(pid), 'watch {}'.format(pid)])
  File "/home/furkan/csc209-w19/grading/mark_a4/marking_scripts/test_helpers.py", line 509, in _exec
    raise e from e
  File "/home/furkan/csc209-w19/grading/mark_a4/marking_scripts/test_helpers.py", line 503, in _exec
    stdout, stderr = proc.communicate(timeout=timeout, input=input_)
  File "/usr/lib/python3.6/subprocess.py", line 843, in communicate
    stdout, stderr = self._communicate(input, endtime, timeout)
  File "/usr/lib/python3.6/subprocess.py", line 1515, in _communicate
    self._check_timeout(endtime, orig_timeout)
  File "/usr/lib/python3.6/subprocess.py", line 871, in _check_timeout
    raise TimeoutExpired(self.args, orig_timeout)
subprocess.TimeoutExpired: Command '['./jobclient8', 'watch 25936', 'watch 25936']' timed out after 10 seconds

======================================================================
ERROR: test_server_watch (__main__.TestServer)
Second client watching.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "server_watch_test.py", line 22, in test_server_watch
    stdoutput, stderror, status = _exec(['./jobclient8', 'watch {}'.format(pid)])
  File "/home/furkan/csc209-w19/grading/mark_a4/marking_scripts/test_helpers.py", line 509, in _exec
    raise e from e
  File "/home/furkan/csc209-w19/grading/mark_a4/marking_scripts/test_helpers.py", line 503, in _exec
    stdout, stderr = proc.communicate(timeout=timeout, input=input_)
  File "/usr/lib/python3.6/subprocess.py", line 843, in communicate
    stdout, stderr = self._communicate(input, endtime, timeout)
  File "/usr/lib/python3.6/subprocess.py", line 1515, in _communicate
    self._check_timeout(endtime, orig_timeout)
  File "/usr/lib/python3.6/subprocess.py", line 871, in _check_timeout
    raise TimeoutExpired(self.args, orig_timeout)
subprocess.TimeoutExpired: Command '['./jobclient8', 'watch 25941']' timed out after 10 seconds

----------------------------------------------------------------------
Ran 2 tests in 22.075s

FAILED (errors=2)
